### ISSUE TITLE
Liftmaster MyQ Garage Door Cover Error

### ISSUE THREAD

**Issue by brianhanifin** [dwainscheeren/lovelace-dwains-theme#1]
It doesn't seem to like my Liftmaster MyQ Garage Door. The Javascript Console gives me the following errors.

```
TypeError: states['cover.garage_door'].attributes.entity_id is undefined
button-card.js line 1485 > Function:16:9
TypeError: entitiesFromGroup is undefined
button-card.js line 1485 > Function:10:23
```

**Comment by dwainscheeren** [comment]
Will look into this asap :)

**Comment by brianhanifin** [comment]
If I put the cover in a group with my front door it works.

**Comment by dwainscheeren** [comment]
This will be fixed in 1.1.2
